HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -5085,14 +5085,14 @@ def self.apply_installation_quality(model, heating_system, cooling_system, unita
     program_calling_manager.addProgram(fault_program)
   end
 
-  # TODO
+  # Calculate delivered cooling q_dot and compressor power p_dot during defrost
   #
-  # @param heat_pump [TODO] TODO
+  # @param heat_pump [HPXML::HeatPump] HPXML Heat Pump object
   # @param unit_multiplier [Integer] Number of similar dwelling units
-  # @param design_airflow [TODO] TODO
-  # @param max_heating_airflow [TODO] TODO
-  # @param fan_watts_per_cfm [TODO] TODO
-  # @return [TODO] TODO
+  # @param design_airflow [Float] Heat pump design air flow rate
+  # @param max_heating_airflow [Float] Maximum heat pump heating air flow rate
+  # @param fan_watts_per_cfm [Float] Heat pump watts per cfm
+  # @return [Array<Float, Float>] Calculated delivered cooling q_dot and compressor power p_dot
```

Comment:
Thanks for filling this out. Please add units (power in W? cooling load in ?? airflow rates in cfm?)